src/qtism/data/AssessmentItem.php

```diff
@@ -228,7 +228,7 @@ public function __construct($identifier, $title, $timeDependent, $lang = '')
      */
     public function setIdentifier($identifier): void
     {
-        if (Format::isIdentifier($identifier, false)) {
+        if (Format::isNormalizedString($identifier)) {
```

Comment:
should we allow spaces in an identifier?

Reply:
It is not forbidden by QTI standard.